spec/support/fakeredis.rb

```diff
@@ -0,0 +1,9 @@
+require "fakeredis/rspec"
+module RedisMemoryDriver
+  def initialize(options = {})
+    options[:driver] = :memory
```

Comment:
Why is it needed?

Reply:
Currently we use fakeredis for the rspec tests and anycable has the driver specified as `ruby`.
https://github.com/anycable/anycable/blob/master/lib/anycable/broadcast_adapters/redis.rb#L33

When adding a new redis driver such as hiredis or fakeredis(:memory) it will just append it to the available drivers array in Redis.  If you create a connection and don't specify the driver it will automatically pick the last on added which is how it was working before.

In anycable-core 1.2.4 we set the default driver for redis as the ruby driver which causes the test suite to fail for graphql-anycable since it wants to use fakeredis.

Reply:
@Envek I've bumped anycable to 1.3.0 and now can specify the :memory driver in the spec_helper to avoid adding the fakeredis support file.